### ISSUE TITLE
no chart appears in MM

### ISSUE THREAD

**Issue by Impatience71** [SvenSommer/MMM-GrafanaChart#11]
Dear supporters, 

I played now several days to show a grafana chart in my MM. I used this use case for creating a grafana chart: 
https://github.com/geerlingguy/internet-pi. The grafana runs in docker with the following parms: 

```
GF_USERS_ALLOW_SIGN_UP=false
GF_INSTALL_PLUGINS=flant-statusmap-panel,ae3e-plotly-panel
GF_ORG_ROLE=Viewer #to just view graphs without authentication
GF_ORG_NAME=Main Org
GF_AUTH_ANONYMOUS_ENABLED=true
GF_AUTH_ANONYMOUS=true
GF_ALLOW_EMBEDDING=true
```

I used the share function in the grafana dashboard to create a url:

`http://pi-hole:3030/d/o9mIe_Aik/internet-connection?from=1624514783736&to=1624601183736&orgId=1&refresh=5m`

and thats my config.js snippet:

```js
{
  disabled: false,
  module: 'MMM-GrafanaChart',
    position: 'top_left',   // This can be any of the regions.
    config: {
       version: "6", // Only add this line if you are using Grafana verison 6 or greater
       id: "o9mIe_Aik", // Only Mandartory if you are using Grafana verison 6 or greater found after /d/ in the url
       host: "http://pi-hole:3030/d/o9mIe_Aik/internet-connection?from=1624514783736&to=1624601183736&orgId=1&refresh=5m", //Mandatory. See url when displaying within grafana
       port: 3030, // Mandatory.
       dashboardname: "internet-connection", // Mandatory.
       orgId: 1, // Mandatory.
       panelId: 2, // Mandatory.
       width: "100%", // Optional. Default: 100%
       height: "100%", // Optional. Default: 100%
       refreshInterval: 900 //Optional. Default: 900 = 1/4 hour
     }
 },
```
But there is nothing to see in the MM. Not even a empty frame. 

Thanks for your comments !
Cheers Peter

**Comment by janre** [comment]
I had the same problem and created a pull-request for it:
https://github.com/SvenSommer/MMM-GrafanaChart/pull/12/files

Probably the path has changed since Grafana version 8. If in the MMM-GrafanaChart.js "/d/" is replaced by "/d-solo/", it works again.

**Comment by SvenSommer** [comment]
Pull request merged. Thank you janre

**Comment by SvenSommer** [comment]
Impatience71  please test your version and report.

**Comment by Bluefish67** [comment]
Hello, is this still maintained? 
I use Grafana 8 and get "Page not found 404 error" when linking.
Here is the config (to be sure all relevant data are XX)
```js
{
         module: 'MMM-GrafanaChart',
                 position: 'top_center',   // This can be any of the regions.
                 config: {
                                version: "8", // Only add this line if you are using Grafana verison 6 or greater
                                id: "9ffefimRz", // Only Mandartory if you are using Grafana verison 6 or greater found after /d/ in the url
                                host: "XXX.XXX.X.XX", //Mandatory. See url when displaying within grafana
                                port: 3000, // Mandatory.
                                dashboardname: "info", // Mandatory.
                                orgId: 1, // Mandatory.
                                panelId: 2, // Mandatory.
                                width: "100%", // Optional. Default: 100%
                                height: "100%", // Optional. Default: 100%
                                scrolling: "no", // Optional. Default: no
                                refreshInterval: 900 //Optional. Default: 900 = 1/4 hour
                        }
},
```
Where could the error be?

**Comment by Bluefish67** [comment]
Ok I think I have found the error. 
I had entered the installed "8" instead of the "6" for version. 
After changing "8" to "6" it is now displayed.

**Comment by bulburDE** [comment]
@Bluefish67 Thanks, that helped me as well. To clarify it a hint in the documentation would be nice that you set it to 6 even if your version is >6. If the change from `/d/` to `/d-solo/` was added on 8.x then this plugin probably won't work with Grafana 6 and 7 anymore since it will look for `/d-solo/` in that cases.

I used the environemnt variables from @Impatience71 in my grafana docker-compose file. To make it work I had to use `GF_SECURITY_ALLOW_EMBEDDING=true`
